actions/singlePicture.js

```diff
@@ -1,9 +1,21 @@
 "use strict";
 
 let pictureServer = require('../services/pictureServer');
+let pictureValidator = require('../middleware/validator');
+
+function updateItem(origObj, dataToUpdate){
```

Comment:
Add lodash as dependencie, and use merge method https://lodash.com/docs/4.17.4#merge